src/apps/material/helper.ts

```diff
@@ -454,6 +457,35 @@ export const useGetHoldings = ({
   return { data, isLoading, isError };
 };
 
+export const getManifestationBasedOnType = (
+  work: Work,
+  materialType: DisplayMaterialType
+): Manifestation => {
+  const { bestRepresentation, all } = work.manifestations;
+
+  // Retrieves the material type of the best representation.
+  const bestRepresentationMaterialType =
+    getManifestationMaterialTypes(bestRepresentation);
```

Comment:
Consider this level of commenting. 

You are basicly writing out in plain english what the following line of code does even though it is not very complex.